lib/ansible/modules/crypto/luks_device.py

```diff
@@ -0,0 +1,448 @@
+#!/usr/bin/python
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+    ---
```

Comment:
I think you need to remove the spaces before `---`; YAML doesn't seem to like that.

Reply:
This (and the whitespace error in line 43) are probably the only things the sanity tests complain about.